packages/discord.js/typings/index.d.ts

```diff
@@ -4309,6 +4310,7 @@ export abstract class MessageManager<InGuild extends boolean = boolean> extends
   public fetch(options: MessageResolvable | FetchMessageOptions): Promise<Message<InGuild>>;
   public fetch(options?: FetchMessagesOptions): Promise<Collection<Snowflake, Message<InGuild>>>;
   public fetchPinned(cache?: boolean): Promise<Collection<Snowflake, Message<InGuild>>>;
+  public forward(reference: Omit<MessageReference, 'type'>): Promise<Message<InGuild>>;
```

Comment:
This feels out of place in the manager...
- we don't have `create`
- we don't have `reply`

So we should either move `create`/`reply` into this manager, or drop this from here altogether. And I'm leaning more towards the former.. thoughts @discordjs/core?

Reply:
If we were to move anything at all, it'd probably just be `create()`. Replying and forwarding can happen here too.